packages/caver-account/src/accountKey/accountKeyLegacy.js

```diff
@@ -41,6 +46,10 @@ class AccountKeyLegacy {
 
     /**
      * Returns an RLP-encoded AccountKeyLegacy string.
+     *
+     * @example
+     * const encoding = accountKeyLegacy.getRLPEncoding()
+     *
      * @return {string}
      */
     // eslint-disable-next-line class-methods-use-this
```

Comment:
Do we need to this comment?

Reply:
Yes, thats for avoiding eslint error for next line